docs/code-examples/point2d_random.rs

```diff
@@ -0,0 +1,39 @@
+//! Log some random points with color and radii.
+use rand::distributions::Uniform;
+use rand::Rng;
+use rerun::components::{ColorRGBA, Point2D, Radius, Rect2D, Vec4D};
+use rerun::{MsgSender, RecordingStreamBuilder};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let (rec_stream, storage) = RecordingStreamBuilder::new("points").memory()?;
+
+    let mut rng = rand::thread_rng();
+    let position_distribs = Uniform::new(-3., 3.);
+
+    let mut positions = vec![];
+    let mut colors = vec![];
+    let mut radii = vec![];
+    for _ in 0..10 {
+        positions.push(Point2D::new(
+            rng.sample(position_distribs),
+            rng.sample(position_distribs),
+        ));
+        colors.push(ColorRGBA::from_rgb(rng.gen(), rng.gen(), rng.gen()));
+        radii.push(Radius(rng.gen()));
+    }
```

Comment:
that's very unidiomatic but I guess it's worth so that we don't have to add `itertools` as a dependency here... eh :shrug:

Reply:
Agree my iter-fu's limitations is showing here...